### PR TITLE
Adding dock support for Brid player

### PR DESCRIPTION
**Pull request by grajzer**

### CODE REVIEW

**Comment by grajzer** [comment]
Thank you for the feedback @alanorozco. Just added several events that were missing in our player (ENDED, AD_START, AD_END, LOADEDMETADATA).

Note that there's a small issue with these dock controls and prerolls. Here's the scenario:

- Clicking play starts video play, preroll is started and AD_START is dispatched.
- User scrolls and player gets into the dock mode -- dock extension sets listeners at this time.
- Because AD_START was dispatched before, controls won't be updated.

Basically, dock extension won't update controls on ad started before player entered the dock mode at least once.
